<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitterMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>941</width>
      <height>551</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="btnSetHeader">
        <property name="text">
         <string>设置表头</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnSetRows">
        <property name="text">
         <string>设置行数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinRowCount"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="btnIniData">
        <property name="text">
         <string>初始化表格数据</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnInsertRow">
        <property name="text">
         <string>插入行</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnAppendRow">
        <property name="text">
         <string>添加行</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="btnDelCurRow">
        <property name="text">
         <string>删除当前行</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btnAutoHeight">
        <property name="text">
         <string>自动调节行高</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnAutoWidth">
        <property name="text">
         <string>自动调节列宽</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="btnReadToEdit">
        <property name="text">
         <string>读取表格内容到文本</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QCheckBox" name="chkBoxTabEditable">
        <property name="text">
         <string>表格可编辑</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="chkBoxRowColor">
        <property name="text">
         <string>间隔行底色</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="chkBoxHeaderH">
        <property name="text">
         <string>显示行表头</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="chkBoxHeaderV">
        <property name="text">
         <string>显示列表头</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QRadioButton" name="rBtnSelectRow">
        <property name="text">
         <string>行选择</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QRadioButton" name="rBtnSelectItem">
        <property name="text">
         <string>单元格选择</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="tableIfno">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>23</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>姓名</string>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>性别</string>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>出生日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>民族</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>分数</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>是否党员</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>0行，0列</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>0行，1列</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>111111</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>1行，0列</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>2行，0列</string>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>5行，0列</string>
       </property>
      </item>
     </widget>
     <widget class="QPlainTextEdit" name="textEdit"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
